ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -489,8 +489,9 @@ private Task<ReactContext> CreateReactContextFromCachedPackagerBundleAsync(Cance
 
             try
             {
-                var reactContext = await CreateReactContextCoreAsync(jsExecutorFactory, jsBundleLoader, token);
-                SetupReactContext(reactContext);
+                var reactContext = await MaybeCallOnDispatcherAsync(() =>
```

Comment:
@rigdern I've narrowed the hack to just this. For some reason, async continuations do not continue on the dispatcher thread in the background. Ideally we can figure out why this is happening so we can eliminate this hack.

Reply:
This might be because the SynchronizationContext isn't set up for the dispatcher yet.

---
In reply to: [145579278](https://github.com/Microsoft/react-native-windows/pull/1450#discussion_r145579278) [](ancestors = 145579278)

Reply:
Yeah, confirmed that `SynchronizationContext.Current == null` in `OnBackgroundActivated`. I have a thread out to find out if this is only configured when the `CoreWindow` is activated.

---
In reply to: [145584686](https://github.com/Microsoft/react-native-windows/pull/1450#discussion_r145584686) [](ancestors = 145584686,145579278)